old-ui/app/components/ens-input.js

```diff
@@ -198,3 +212,15 @@ EnsInput.prototype.ensIconContents = function (recipient) {
 function getNetworkEnsSupport (network) {
   return Boolean(networkMap[network])
 }
+
+function getNetworkRnsSupport (network) {
+  if (network == RSK_CODE) {
+    return RNSRegistryData.address.rskMainnet;
+  }
+  
+  if (network == RSK_TESTNET_CODE) {
+    return RNSRegistryData.address.rskTestnet;
+  };
+
+  return;
+}
```

Comment:
For return type consistency with the existing `getNetworkEnsSupport` implementation:

```suggestion
function getNetworkRnsSupport (network) {
  return (network == RSK_CODE || network == RSK_TESTNET_CODE);
}
```

Alternatively, consider creating an an `networkMapRns` object and use it in the same manner as `networkMap`.

Reply:
The `networkMap` belongs to another package (https://github.com/ethjs/ethjs-ens/blob/master/lib/network-map.json). Anyway, just added another function to improve the code readability

Reply:
Hmm, maybe there should be the equivalent network map added to `@rsksmart/rns-registry`.
(but that's out of scope of this PR!)